src/views/TodoList.vue

```diff
@@ -4,28 +4,15 @@
       <thead>
         <tr>
           <td>
-            <input
-              type="radio"
-              checked="checked"
-              name="workingStatus"
-              v-model="jobStatus"
-              value="全て"
+            <input type="radio" checked="checked" name="workingStatus" v-model="jobStatus" value="全て"
```

Comment:
以下のようにも記述できますので、覚えておきましょう。
```suggestion
            <input type="radio" name="workingStatus" v-model="jobStatus" value="全て" checked
```

Reply:
ありがとうございます！